routes/users.js

```diff
@@ -242,12 +242,13 @@ router.post("/login", async (req, res) => {
 });
 
 // Register Handle
-router.post("/register", async (req, res) => {
-  const { name, email, password, location } = req.body;
-  console.log({ name, email, password, location })
 
-  //check required fields
+// Register Handle
+router.post("/register", async (req, res) => {
+  const { name,email,password,location} = req.body;
+  // check required fields
   if (!name || !email || !password || !location) {
+    console.log(name, email, password, location)
```

Comment:
if there is no name, email, password or location - there will be 'undefined' in your console.log
I believe that will be better to use it in another place for this log, but this is not critical

in case if user has no credentials - better send to the console another message

Reply:
removed console.log